modules/gcs/main.tf

```diff
@@ -32,6 +32,9 @@ resource "google_storage_bucket" "bucket" {
   versioning {
     enabled = var.versioning
   }
+  autoclass {
```

Comment:
We typically use dynamic blocks even for booleans, so we avoid definining the block when false. It's not hugely important, but a convention we followed so far and it would have avoided having to modify test results. It's fine to leave it like this, we might refactor it later.